distributions/docker/solr/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM solr:8.1.1
+
+COPY maven/replication-solr-managed-schema.xml /opt/solr/server/solr/configsets/_default/conf/managed-schema
+
+ENTRYPOINT  bash -c \
```

Comment:
not necessary right now but we should look at wrapping this all in a script instead of a complex entrypoint command that calls out to `bash -c`